partner_event/models/partner_action.py

```diff
@@ -0,0 +1,154 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#    This module copyright (C) 2015 Savoir-faire Linux
+#    (<http://www.savoirfairelinux.com>).
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+from openerp import models, fields, api
+
+
+class PartnerAction(models.Model):
+    _name = 'partner.action'
+    _description = 'Partner Action'
+    _order = "priority"
+
+    # Getters/Setters
+    def _get_done(self):
```

Comment:
hey vincent!

You might want to use @api.multi or @api.one to avoid to use the loop. Sadly, I can't test for you, and I never remember which one to use in this case :)